### PR TITLE
Updated Kotlin example to 1.0.0 beta

### PR DESCRIPTION
**Pull request by cmelchior**
Related to #1647 

Updates our Kotlin example to be compatible with the latest Kotlin 1.0.0-beta 

@realm/java

### CODE REVIEW

**Comment by zaki50** [comment]
:+1:

**Comment by dalinaum** [comment]
:+1:

**Comment by zaki50** [comment]
@cmelchior one more thing 
He said following configuration needed in build.gradle.

I'm confirming now.

```
kapt {
    generateStubs = true
}
```

**Comment by cmelchior** [comment]
It doesn't seem to be a problem in our case. It is described here: http://blog.jetbrains.com/kotlin/2015/06/better-annotation-processing-supporting-stubs-in-kapt/

As far as I understand it you only need that if you have Kotlin code that references classes generated by the annotation processor.

**Comment by dalinaum** [comment]
device-2015-10-25-030047

`Number of persons` of `basic Query operation` and `basic link Query operation` are `0`. It seems meaningless tests.

**Comment by dalinaum** [comment]
Should this PR update `changelog.txt`?

**Comment by cmelchior** [comment]
It probably should, yes. Didn't notice the output was wrong, will take another look

**Comment by dalinaum** [comment]
It looks to me like we can remove `field:` after merging #1663

**Comment by cmelchior** [comment]
Yes

**Comment by cmelchior** [comment]
even before, because I don't use @Required in the example project

**Comment by cmelchior** [comment]
I updated the example so it no longer uses `@field:`. Also rearranged some code so the output to the screen makes more sense.

**Comment by kneth** [comment]
It looks fine to me (never really tried Kotlin).

File: examples/kotlinExample/src/main/kotlin/io/realm/examples/kotlin/KotlinExampleActivity.kt [hunk 1]
```diff
@@ -49,6 +49,10 @@ public class KotlinExampleActivity : Activity() {
         // These operations are small enough that
         // we can generally safely run them on the UI thread.
 
+        // Create configuration and reset Realm.
+        val config = RealmConfiguration.Builder(this).build();
```
Comment (nhachicha) [inline examples/kotlinExample/src/main/kotlin/io/realm/examples/kotlin/KotlinExampleActivity.kt]: No need for semicolons in Kotlin 👍

File: examples/kotlinExample/src/main/kotlin/io/realm/examples/kotlin/KotlinExampleActivity.kt [hunk 2]
```diff
@@ -49,6 +49,10 @@ public class KotlinExampleActivity : Activity() {
...
+        Realm.deleteRealm(config);
```
Comment (nhachicha) [inline examples/kotlinExample/src/main/kotlin/io/realm/examples/kotlin/KotlinExampleActivity.kt]: same

**Pull request merged**